airbyte-integrations/connectors/destination-s3/Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM airbyte/integration-base-java:dev
 WORKDIR /airbyte
 ENV APPLICATION destination-s3
 
-ADD build/distributions/${APPLICATION}*.tar /airbyte
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+
+RUN tar xf ${APPLICATION}.tar --strip-components=1
```

Comment:
```suggestion
ADD build/distributions/${APPLICATION}*.tar /airbyte
```

Reply:
This change ruins integration tests.
https://github.com/airbytehq/airbyte/runs/4570205306?check_suite_focus=true
I removed it back.